roles/ceph-common/tasks/main.yml

```diff
@@ -1,55 +1,43 @@
 ---
-- name: include installs/install_on_redhat.yml
-  include: installs/install_on_redhat.yml
+- include_role:
```

Comment:
Shouldn't this be `include_tasks` or `import_tasks` instead? You can read https://docs.ansible.com/ansible/2.4/playbooks_reuse_includes.html

Reply:
> Shouldn't this be include_tasks or import_tasks instead?

This task has [`static` option set to `False`](https://github.com/ceph/ceph-ansible/pull/3016/files/19d13314c96f9a92e0096c154165c7ba77febe1d#diff-fbf6d60f8ff9940f0621a8a5c45c392eL9) so `include_task` is correct. Quoting [docs](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse.html#dynamic-vs-static) -

```
If you use any import* Task (import_playbook, import_tasks, etc.), it will be static. If you use any include* Task (include_tasks, include_role, etc.), it will be dynamic.
```

Reply:
Ok let's change it to `include_tasks` then.